app/models/hardware.py

```diff
@@ -0,0 +1,30 @@
+from typing import Union
+from uuid import uuid4
+
+from sqlalchemy import Column, String
+
+from app import wrapper
+
+
+class Hardware(wrapper.Base):
+    """
+    This is the hardware-model for cryptic-game
+    """
+
+    __tablename__: str = "hardware"
+
+    uuid: Union[str, Column] = Column(String(36), primary_key=True, unique=True)
+    device_uuid: Union[str, Column] = Column(String(36))
+    hardware_element: Union[str, Column] = Column(String(40))
+    hardware_typ: Union[str, Column] = Column(String(36))
+
+    @staticmethod
+    def create(device: str, hardware: str, hardware_typ: str) -> "Hardware":
+        hd: Hardware = Hardware(
```

Comment:
Try to use talking variables like `hardware` instead of `hd`

Reply:
done